### PR TITLE
zarik staging

### PR DESCRIPTION
**Pull request by zarik5**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+* Shallow rename "client" -> "device" (by @zarik5 #2120)
+* Update manifest for AppLab (by @zarik5 #2146)
+* Allow recentering in the lobby (by @zarik5 #2155)
+* Enable recentering in the lobby room (by @zarik5 #2155)
+* Tweak graph labels and colors (by @zarik5 #2176)
+* Fix controller detection on Pico Neo 3 Link (by @zarik5 #2192)
+* Loosen restrictions on device selection with VoiceMeeter (by @xuan25 #2209)
+* Fix HEVC black screen on Linux/VAAPI (by @Nibor62 #2203)
+* Show hands and controllers in the lobby (by @zarik5 #2218)
+* Add PipeWire support on Linux (by @Meister1593 #1973)
```
Comment (The-personified-devil) [inline CHANGELOG.md:21]: That's missing the Pico 3 Link stuff or is that intentional

**Review by The-personified-devil — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zarik5) [reply CHANGELOG.md:21]: No, it's there. I've put the entries in chronological order on when it was merged

Comment (zarik5) [reply CHANGELOG.md:21]: I messed up another thing instead lol

Comment (The-personified-devil) [reply CHANGELOG.md:20]: yea it's credited to you and the PR is from Holo

**Review by The-personified-devil — APPROVED** [review]

**Pull request merged**